src/pages/students.tsx

```diff
@@ -2,19 +2,12 @@ import React from "react";
 import MyStudents from "../components/case_manager/MyStudents";
```

Comment:
Please move this route to `/pages/students/index.tsx`.

Reply:
Oops this looks confusing because of where I commented. Don't change the import, but move `src/pages/students.tsx` -> `src/pages/students/index.tsx`.